x-pack/plugins/security/server/audit/audit_events.ts

```diff
@@ -173,6 +173,32 @@ export function userLogoutEvent({
   };
 }
 
+export function userSessionConcurrentLimitLogoutEvent({
+  username,
+  provider,
+  userProfileId,
+}: UserLogoutParams): AuditEvent {
+  return {
+    message: `User [${username}] is logging out due to exceeded concurrent sessions limit for ${provider.type} provider [name=${provider.name}]`,
+    event: {
+      action: 'user_logout',
```

Comment:
__note:__ I was initially considering using a dedicated `user_logout_concurrent_session` action, but didn't find enough arguments to separate it from the normal logout in audit logs, but let me know what you think.

Reply:
I think this makes sense since we're also not currently distinguishing between user initiated logout events and logouts triggered by session timeouts.